x-pack/plugins/security_solution/common/api/entity_analytics/asset_criticality/get_asset_criticality.schema.yaml

```diff
@@ -3,28 +3,28 @@ info:
   version: 1.0.0
```

Comment:
Could you elaborate why version `1.0.0` is used?

As far as I see only Risk Score and Asset Criticality APIs use `1.0.0` version. The rest Security Solution APIs use version `1`. Bundling public + internal API's produces 3 bundles instead of two

![image](https://github.com/elastic/kibana/assets/3775283/0958bbe4-521c-4dbe-9dc7-83868fb1f8dc)

For consistency reasons it's preferable to use the same version strategy for internal APIs.

Reply:
All Entity Analytics `yaml` files are old and were generated with version `1.0.0`.  Should I update all private APIs to use version `1`?

Reply:
@machadoum,

Version `2023-10-31` is used only for public APIs. APIs registered at `router` with `access: 'public'`. All Entity Analytics APIs are internal at the moment. So it should use versioning model for internal APIs. I haven't seen any restrictions on internal APIs versioning but existing OAS describing Security Solution internal APIs use version `1` (besides Entity Analytics), for example [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/common/api/detection_engine/rule_monitoring/rule_execution_logs/get_rule_execution_events/get_rule_execution_events_route.schema.yaml#L4).